db/job.go

```diff
@@ -581,6 +583,39 @@ func (j *job) CreateBuild() (Build, error) {
 	return build, nil
 }
 
+func (j *job) ClearTaskCache(stepName string, cachePath string) (int64, error) {
+	tx, err := j.conn.Begin()
+	if err != nil {
+		return -1, err
```

Comment:
It's more common in Go to return zero-values in err cases (so `0` instead of `-1`).